feat(locale): add Tajik (тоҷикӣ) language support

lb18254983179 opened the pull request:
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.
- Added comprehensive Tajik (тоҷикӣ) translations for Element Plus components.
- Follows existing locale file structure.

lb18254983179 committed:
feat(locale): add Tajik (тоҷикӣ) language support

lb18254983179 committed:
feat(locale): add Tajik (тоҷикӣ) language support

lb18254983179 committed:
Merge branch 'dev' into dev

lb18254983179 commented:
Copy and paste zh-cn.ts, rename it tajik.ts, and translate the content into Tajik language.

plainheart commented on packages/locale/lang/tajik.ts:
```diff
@@ -0,0 +1,179 @@
+export default {
+  name: 'tg', // 塔吉克语
```
Please rename this file to `tg.ts` for uniform naming style.

lb18254983179 replied:
Please rename this file to tg.ts for uniform naming style.

plainheart commented on packages/locale/index.ts:
```diff
@@ -65,6 +65,7 @@ export { default as zhCn } from './lang/zh-cn'
 export { default as zhTw } from './lang/zh-tw'
 export { default as zhHk } from './lang/zh-hk'
 export { default as zhMo } from './lang/zh-mo'
+export { default as tg } from './lang/tajik'
```
Rename `tajik` to `tg` and order this line by dictionary.

lb18254983179 replied:
Please rename this file to tg.ts for uniform naming style.

lb18254983179 committed:
fix(locale): Rename tajik to tg and order this line by dictionary. Please rename this file to tg.ts for uniform naming style.

lb18254983179 committed:
Rename tajik to tg and order this line by dictionary.Please rename this file to tg.ts for uniform naming style.

Merge branch 'dev' of https://github.com/lb18254983179/element-plus into dev

lb18254983179 commented:
Code has been changed and submitted to github.

lb18254983179 commented:
Hello, I have renamed the Tajik ts file and modified the import location. I have submitted the code to GitHub for review. Thank you.



---Original---
From: "Zhongxiang ***@***.***&gt;
Date: Sat, Aug 2, 2025 14:14 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [element-plus/element-plus] feat(locale): add Tajik (тоҷикӣ) language support (PR #21551)




 
@plainheart requested changes on this pull request.
 
 
In packages/locale/lang/tajik.ts:
 &gt; @@ -0,0 +1,179 @@ +export default { +  name: 'tg', // 塔吉克语  
Please rename this file to tg.ts for uniform naming style.
 
 
In packages/locale/index.ts:
 &gt; @@ -65,6 +65,7 @@ export { default as zhCn } from './lang/zh-cn'  export { default as zhTw } from './lang/zh-tw'  export { default as zhHk } from './lang/zh-hk'  export { default as zhMo } from './lang/zh-mo' +export { default as tg } from './lang/tajik'  
Rename tajik to tg and order this line by dictionary.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

lb18254983179 committed:
fix(locale): Submission after pnpm format

lb18254983179 commented:
I have successfully run the pnpm format command and submitted the code.

lb18254983179 committed:
Merge branch 'dev' into dev

lb18254983179 committed:
Merge branch 'dev' into dev

lb18254983179 committed:
Merge branch 'dev' into dev
